ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java

```diff
@@ -256,15 +257,15 @@ private PerfTest save(PerfTest perfTest) {
 		checkNotNull(perfTest);
 		// Merge if necessary
 		if (perfTest.exist()) {
-			PerfTest existingPerfTest = perfTestRepository.findOne(perfTest.getId());
-			perfTest = existingPerfTest.merge(perfTest);
+			Optional<PerfTest> existingPerfTest = perfTestRepository.findOne(idEqual(perfTest.getId()));
+			existingPerfTest.ifPresent(perfTest1 -> perfTest1.merge(perfTest));

```

Comment:
please check if the perfTest here is always retrieved from DB.
if so.. I think we don't need merge here.